assets/src/components/map/markers/stopMarker.tsx

```diff
@@ -0,0 +1,305 @@
+    "stroke-transparent",
+    "c-stop-icon-focus_stroke-eggplant-500",
+  ]
+  return (
+    <circle
+      {...props}
+      className={joinClasses([className, ...focusCircleClasses])}
+    ></circle>
+  )
+}
+
+export const StopIcon = ({
+  type,
+  selected = false,
+}: {
+  type: StopType
+  selected?: boolean
+}) => {
+  const commonSvgClasses = [
+    "d-block",
+    "overflow-visible",
+    "transform-origin-center",
+  ]
+  const embeddedSvgClasses = joinClasses(commonSvgClasses)
+
+  const circleClasses = [
+    // Defaults
+    "transform-origin-center",
+    "fill-white",
+    "stroke-gray-700",
+    "stroke-width-1",
+
+    // Interactions
+    "c-stop-icon-selected_stroke-eggplant-700",
+    "c-stop-icon-selected_fill-eggplant-200",
+
+    "c-stop-icon-hover_stroke-width-2",
+    "c-stop-icon-focus_stroke-width-2",
+    "c-stop-icon-selected_stroke-width-2",
+  ]
+  const strokeWidthOffset = 3
+
+  const busClasses = joinClasses([
+    // Defaults
+    "scale--2",
+    "fill-gray-700",
+    "stroke-width-0",
+    "transform-origin-center",
+
+    // Interactions
+    "c-stop-icon-hover_fill-eggplant-700",
+    "c-stop-icon-focus_fill-eggplant-700",
+    "c-stop-icon-selected_fill-eggplant-700",
+  ])
+
+  return (
+    <svg
+      // Size to container
+      width="100%"
+      height="100%"
+      className={joinClasses([
+        ...commonSvgClasses,
+        "c-stop-icon", // Interaction Group
+
+        // Interactions
+        "transition-2",
+        "scale-2-hover",
+        "scale-2-focus",
+        "scale-2-selected",
+      ])}
+      data-selected={selected || null} // Remove attribute if `false`
+    >
+      {/* `width` and `height` is not specified on child SVG's so they size to
+       * the parent SVG. This makes `StopIconSizeFromStopType` the source of
+       * truth for the size on the map.
+       * Specifying the `viewbox` ensures that the aspect ratio is correct,
+       * and Leaflet specifies the container size on the map via `DivIconOptions.iconSize`,
+       * which the child SVG's fill because `width` and `height` are `auto` by
+       * default, and the parent SVG is explicitly setting `width` and `height`
+       * to `100%`
+       */}
+      {/*
+        It does suck that we're completely replacing the icon when the type
+        changes, because that breaks any animations or states that were applied
+        to the previous element, but in trying to stay as close to the provided
+        art assets as possible, this was easiest.
+       */}
+      {type === StopType.Small && (
+        <svg className={embeddedSvgClasses} viewBox="0 0 14 14" fill="none">
+          <FocusCircle cx="7" cy="7" r={5 + strokeWidthOffset} />
+          <circle
+            className={joinClasses([
+              ...circleClasses,
+              "c-stop-icon-hover_stroke-eggplant-700",
+            ])}
+            cx="7"
+            cy="7"
+            r="5"
+            fill="white"
+            stroke="#3C3F4C"
```

Comment:
**suggestion:** it looks like this is actually the `$color-gray-700` from our design system, could you use a utility class instead of setting the value manually?

Reply:
I left it because it was part of the figma export, but I'll add the classes(to all SVG's that match this same issue) 👍🏻.

If we have the classes, should we still have the fill and stroke as fallbacks because they were part of the export, or do you still feel it should be dropped?

Reply:
Oh turns out they already have the classes specified. So, remove the attributes?

Reply:
Turns out that we could also specify `clip-rule` and `fill-rule` via CSS

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-rule
> Note: As a presentation attribute, fill-rule can be used as a CSS property.

Thoughts on generating the CSS utilitiy classes and also specifing these properties via CSS?

Reply:
> Turns out that we could also specify `clip-rule` and `fill-rule` via CSS
> 
> https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-rule
> 
> > Note: As a presentation attribute, fill-rule can be used as a CSS property.
> 
> Thoughts on generating the CSS utilitiy classes and also specifing these properties via CSS?

I can't immediately think of any downsides - if you're okay with it then so am I.

Reply:
a1b4d00c aceb53c6